conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/ImplementationGuide-hl7.fhir.us.davinci-hrex.json

```diff
@@ -58,7 +58,7 @@
 			"id": "uscore",
 			"uri": "http://hl7.org/fhir/us/core/ImplementationGuide/hl7.fhir.us.core",
 			"packageId": "hl7.fhir.us.core",
-			"version": "4.0.0"
+			"version": "3.1.1"
```

Comment:
Discussed as a team... it should be 3.1.1 consistently through this PR.

Reply:
done